.github/workflows/build_and_test.yml

```diff
@@ -33,29 +33,19 @@ concurrency:
 env:
   BUILD_ROOT: ${{ github.workspace }}/bazel-bin/build
   KONG_TEST_COVERAGE: ${{ inputs.coverage == true || github.event_name == 'schedule' }}
+  RUNNER_COUNT: 10
```

Comment:
Let’s not raise this yet. We share OSS runners with other repo under the org and increasing it previously has caused starvation issues to other teams. We should keep it at the same concurrency as we do now (7 runners).